runtime/src/chpl-gpu.c

```diff
@@ -42,11 +42,11 @@ void chpl_gpu_on_std_modules_finished_initializing(void) {
   chpl_gpu_impl_on_std_modules_finished_initializing();
 }
 
-void chpl_gpu_launch_kernel(int ln, int32_t fn,
-                            const char* fatbinData, const char* name,
-                            int grd_dim_x, int grd_dim_y, int grd_dim_z,
-                            int blk_dim_x, int blk_dim_y, int blk_dim_z,
-                            int nargs, ...) {
+inline void chpl_gpu_launch_kernel(int ln, int32_t fn,
```

Comment:
Did you see inlining this make a performance impact?

Don't worry about it if you don't know off the top of your head (it seems like a prime candidate for inlining), I'm just curious.

Reply:
I did see some impact in an earlier iteration of this PR. It was small but larger than noise.

I thought about removing while I was polishing the branch, but wanted to keep it in. As you said, it should probably be inlined.